docs/docs-ref-autogen/outlook/office.body.yml

```diff
@@ -121,6 +121,21 @@ items:
 
       <tr><td>[Applicable Outlook
       mode](https://docs.microsoft.com/outlook/add-ins/#extension-points)</td><td>Compose</td></tr></table>
+
+      #### Examples
+
+
+      ```javascript
+
+      Office.context.mailbox.item.body.getTypeAsync(function (asyncResult) {
+          if (asyncResult.status == "failed") {

```

Comment:
I think this one should be === otherwise status can be any truthy value, which could be weird unless status is also very rigidly set